lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -321,6 +331,28 @@ newStakePoolLayer nl db@DBLayer {..} worker = do
             rewards = view (#metrics . #nonMyopicMemberRewards) . stakePool
             (randomWeight, stakePool) = (fst, snd)
 
+    _forceMetadataGC :: TVar ThreadId -> IO ()
+    _forceMetadataGC tvTid =
+        -- We force a GC by resetting last sync time to 0 (start of POSIX
+        -- time) and have the metadata thread restart.
+        bracketSyncThread tvTid (atomically (putLastMetadataGC (fromIntegral @Int 0)))
```

Comment:
**Request:** Wrap this long expression (see [line limit](https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters)).

```suggestion
        bracketSyncThread tvTid
            $ atomically
            $ putLastMetadataGC
            $ fromIntegral @Int 0
```